gtsfm/retriever/rig_retriever.py

```diff
@@ -0,0 +1,98 @@
+"""Retriever for camera rigs that come with a constraints file.
+
+Author: Frank Dellaert
+"""
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Tuple
+
+import gtsam
+import numpy as np
+from gtsfm.loader.loader_base import LoaderBase
+from gtsfm.retriever.retriever_base import RetrieverBase
+
+
+@dataclass
+class Constraint:
+    a: int = 0
+    b: int = 1
```

Comment:
Should we skip default values for a and b?

Reply:
done